Roadmap.md

```diff
@@ -1,40 +1,47 @@
 
 Please file [issues](https://github.com/Microsoft/MixedRealityToolkit-Unity/issues) if you encounter issues with specific versions of Unity.
 
-## Release cadence
-The master branch is on a monthly release cadence (ex: 2017.2.1.4 in April 2018, 2017.4.0.0 in May 2018, etc.). Precice timing within the month may vary based on planned features and number of issues being addressed.
+### Release cadence
 
-As needed, patch and hot fix releases will be released between the sceduled cadence.
+The master branch is on a quarterly release cadence (ex: 2017.4.2 in September 2018, 2017.4.3 in Q1 2019, etc.). Precise timing within the month may vary based on planned features and number of issues being addressed.
+
+As needed, patch and hot fix releases will be released between the scheduled cadence.
 
 ### Note on version numbers
+
 Starting with the May 2018 release, the MRTK version numbers are adopting [Semantic versioning](https://semver.org/) rather than aligning with a specific Unity release.
 
 The MRTK will use the first two version fields (2017.4) to indicate the recommended version is the Unity 2017 LTS version. The third field will indicate the MRTK revision number (ex: 2017.4.2). The fourth will always be zero (2017.4.2.0).
 
-## Development branches
-For the [master](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/master) branch releases, there is a unique developement / stabilization branch for each release. Please refer to the [Upcoming releases table](#upcoming-releases), or the [Projects](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects) page for specifics.
+### Development branches
+
+For the [master](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/master) branch releases, development is being done in [htk_development](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/htk_development). Please refer to the [Upcoming releases table](#upcoming-releases), or the [Projects](https://github.com/Microsoft/MixedRealityToolkit-Unity/projects) page for specifics.
 
 Please see [Branch management plan](#branch-management-plan) for more details.
 
+## MRTK Version Next
 
-# MRTK Version Next
-MixedRealityToolkit Version Next aims to further extend the capabilities of the toolkit and also introduce new features, including the capability to support more VR/AR/XR platforms beyond Microsoft's own Windows Mixed Reality platform.
+MixedRealityToolkit Version Next aims to further extend the capabilities of the toolkit and also introduce new features, including the capability to support more VR / AR / XR platforms beyond Microsoft's own Windows Mixed Reality platform.
```

Comment:
This might be a silly request but I think the order of the realities is important:

` AR / XR / VR ` to reflect the spectrum better.

Reply:
Do you see XR as a segment of the spectrum, rather than the entire spectrum itself?

Reply:
Usually I refer to the whole spectrum as Mixed Reality not XR.

Reply:
I've always read XR as "<Wildcard> Reality", thus the whole spectrum. But you know, we're all making this stuff up as we go, that's the fun part. :)

Reply:
I also tend to use MR instead of XR.

I updated the order of this list.

Reply:
I only see spaces your commit. :D